src/store/flow/block-types/BaseBlock.ts

```diff
@@ -48,19 +48,36 @@ export const actions = {
       vendor_metadata: await dispatch('initiateExtraVendorConfig'),
     },
     )
+
+    // Define exits after we have the whole final props, this is important for dynamic test value
+    if (props?.exits === undefined) {
```

Comment:
I removed all exit generation within each block's `creatWith()` in favour of this new block of codes